addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1387,6 +1392,50 @@ export class ListRenderer extends Component {
                     toFocus = buttons[index - 1] || currentButton;
                 }
                 break;
+            case "shift+arrowdown": {
+                const recordIndex = this.props.list.records.indexOf(record);
+                if (recordIndex >= this.props.list.records.length - 1) {
+                    return false;
+                }
+                const nextRecord = this.props.list.records[recordIndex + 1];
+
+                const isExpanding = this.firstArrowShiftRecord.index < recordIndex + 1;
+                if (isExpanding) {
+                    nextRecord.toggleSelection(this.firstArrowShiftRecord.record.selected);
+                } else {
+                    record.toggleSelection(false);
+                }
+                this.lastCheckedRecord = nextRecord;
+                toFocus = this.findFocusFutureCell(cell, cellIsInGroupRow, "down");
+                break;
+            }
+            case "shift+arrowup": {
```

Comment:
What about "shift+pageup" and "shift+pagedown" ?

This would be nice to have as well ;-)

Reply:
It's true that it could be a nice feature 👍 . I think it should be specified and be part of a new task

Reply:
@bouvyd what do you think ?